earth_enterprise/src/server/wsgi/common/postgres_manager.py

```diff
@@ -125,7 +127,17 @@ def Query(self, query, parameters=None):
 
     with self._Cursor() as cursor:
       self._logger.debug(cursor.mogrify(query, parameters))
-      cursor.execute(query, parameters)
+      try:
+        cursor.execute(query, parameters)
+      except psycopg2.ProgrammingError as exc:
+        print exc.message
```

Comment:
when are we hoping to merge this? before or after upgrading to python 3.8, or concurrently?

Reply:
I think before, but it is an experimental feature, so its not critical ... @tst-ccamp